CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+2.1.1 (In Progress)
+===================
+
+Bug Fixes
+---------
+
+- Ignoring a call will now properly stop the ringing sound
+- NPM versioning has been fixed to specify >=12 rather than exactly 12
```

Comment:
Is this change included in this PR?

Reply:
This was added on master and the CHANGELOG wasn't updated, so just adding it here.